WordPress/src/main/res/values/strings.xml

```diff
@@ -752,6 +752,9 @@
     <string name="reader_toast_err_get_blog_info">Unable to show this blog</string>
     <string name="reader_toast_err_already_follow_blog">You already follow this blog</string>
     <string name="reader_toast_err_follow_blog">Unable to follow this blog</string>
+    <string name="reader_toast_err_follow_blog_not_found">This blog could not be found</string>
+    <string name="reader_toast_err_follow_blog_not_authorized">You are not authorized to access this blog</string>
+    <string name="reader_toast_err_follow_blog_moved">This blog no longer exists</string>
```

Comment:
Shouldn't this error be something like "This blog no longer exists at the address you've provided", or something similar?